test/modules/auth/components/BvvPlusPasswordCredentialFooter.test.js

```diff
@@ -0,0 +1,50 @@
+import { $injector } from '../../../../src/injection';
+import { BvvPlusPasswordCredentialFooter } from '../../../../src/modules/auth/components/BvvPlusPasswordCredentialFooter';
+import { TestUtils } from '../../../test-utils';
+window.customElements.define(BvvPlusPasswordCredentialFooter.tag, BvvPlusPasswordCredentialFooter);
+
+describe('BvvPlusPasswordCredentialFooter', () => {
+	const setup = async (state = {}) => {
+		const initialState = {
+			notifications: {
+				notification: null
+			},
+			media: {
+				portrait: false
+			},
+			...state
+		};
+		TestUtils.setupStoreAndDi(initialState, {});
+
+		$injector.registerSingleton('TranslationService', { translate: (key) => key });
+
+		return TestUtils.render(BvvPlusPasswordCredentialFooter.tag);
+	};
+
+	describe('when instantiated', () => {
+		it('has a model containing default values', async () => {
+			await setup();
+			const model = new BvvPlusPasswordCredentialFooter().getModel();
+
+			expect(model).toEqual({});
+		});
+	});
+	describe('when footer is rendered', () => {
+		it('displays the information text', async () => {
+			const element = await setup();
+			expect(element.shadowRoot.textContent).toContain('auth_passwordCredentialPanel_footer_register_for_role');
+			expect(element.shadowRoot.textContent).toContain('auth_passwordCredentialPanel_footer_register_information');
+		});
```

Comment:
Let's improve the tests with some more specific tests:
- `auth_passwordCredentialPanel_footer_register_for_role` must be a child of `.footer_register`
- add a test for the correctly configured `ba-badge` component:
```
$injector.registerSingleton('TranslationService', { translate: (key, params) => html`key ${params[0]}` });
...

expect(element.shadowRoot.querySelector('ba-badge').label).toBe('Plus');
expect(element.shadowRoot.querySelector('ba-badge').color).toBe('var(--text3)');
...etc

```